docs/api/index.rst

```diff
@@ -11,12 +11,18 @@ API
 Changelog
 ---------
 
+1.11 (unreleased)
+'''''''''''''''''
+
+- Parent metadata is now readable if children creation is allowed
+- Now returns a ``412`` instead of a ``403`` if the ``If-None-Match: *`` header is
+  provided and the ``create`` permission is allowed
```

Comment:
I believe `test_list_can_be_obtained_if_allowed_to_create` and `test_list_is_denied_if_not_allowed_to_create` cover this (?)